pom.xml

```diff
@@ -59,6 +59,12 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```

Comment:
is this required? what is this needed for?

Reply:
It disables the deployment for the aggregator project

Reply:
thanks!